tests/rptest/services/redpanda.py

```diff
@@ -192,7 +195,8 @@ class RedpandaService(Service):
     LOG_LEVEL_KEY = "redpanda_log_level"
     DEFAULT_LOG_LEVEL = "info"
 
-    SUPERUSER_CREDENTIALS = ("admin", "admin", "SCRAM-SHA-256")
+    SUPERUSER_CREDENTIALS: SaslCredentials = SaslCredentials(
```

Comment:
nit: tools like pyright seem to generally be pretty good at deducing the type from assignments.